src/icrar/leap-accelerate/common/eigen_3_3_beta_1_2_support.h

```diff
@@ -1,28 +0,0 @@
-/**
- * ICRAR - International Centre for Radio Astronomy Research
- * (c) UWA - The University of Western Australia
- * Copyright by UWA(in the framework of the ICRAR)
- * All rights reserved
- *
- * This library is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 2.1 of the License, or (at your option) any later version.
- *
- * This library is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public
- * License along with this library; if not, write to the Free Software
- * Foundation, Inc., 59 Temple Place, Suite 330, Boston,
- * MA 02111 - 1307  USA
- */
-
-// Eigen3.3 beta1-2 uses an unsupported cuda macro, the following code redefines __CUDACC_VER__.
-// Ensure this header is included before all eigen headers.
-#ifdef __CUDACC_VER__
-#undef __CUDACC_VER__
-#define __CUDACC_VER__ ((__CUDACC_VER_MAJOR__ * 10000) + (__CUDACC_VER_MINOR__ * 100))
```

Comment:
Currently all builds are using a prerelease version of eigen that's compatible with the latest cuda, so this can be removed